pvacseq/lib/main.py

```diff
@@ -93,21 +97,44 @@ def main(args_input = sys.argv[1:]):
 #            ).stdout.decode().split('\n')
 #        )
 
+    tmp_dir = tempfile.TemporaryDirectory()
+    tmp_split_fasta_prefix = os.path.join(tmp_dir.name, args.sample_name + "_" + str(args.peptide_sequence_length) + ".fa.split")
+
+    run(["split --lines=400 %s %s.split" % (fasta_file_path, tmp_split_fasta_prefix)], shell=True)
```

Comment:
Good catch. I actually ran into this problem before but just kinda ignored it. Setting the `check=True` flag is definitely a good idea.

One the top of my head I believe that the file names could contain `*`, `:`, and `-` as part of the allele name. I'm not sure if those would cause problems but I guess it would be better to not us `shell=True`. 

I'll push a commit to fix this shortly.